Assets/MapzenMap.cs

```diff
@@ -78,7 +82,28 @@ public void DownloadTiles()
                 float offsetX = (tileAddress.x - bounds.min.x);
                 float offsetY = (-tileAddress.y + bounds.min.y);
 
-                TileTask task = new TileTask(tileAddress, groupOptions, response.data, offsetX, offsetY, regionScaleRatio);
+
+                float unitConverter = 1.0f;
+                switch (regionScaleUnit)
```

Comment:
We could wrap this bit of code directly under `RegionScaleUnits.cs`. We can do a cleanup PR when updating the UI interface for this too.